services/registrations/EmailService/SESEmailService.js

```diff
@@ -69,12 +84,11 @@ export default class SESEmailService {
       logoBase64
     };
     const rawHtml = user.isPartner ?
-      getPartnerCalendarInviteTemplate(emailParams)
-      : event.isApplicationBased ?
-        getDefaultPaymentProcessedTemplate(emailParams)
-        : getDefaultCalendarInviteTemplate(emailParams);
+      getPartnerCalendarInviteTemplate(emailParams) :
+      event.isApplicationBased ?
+      getDefaultPaymentProcessedTemplate(emailParams) :
+      getDefaultCalendarInviteTemplate(emailParams);
```

Comment:
Not in scope for this PR but seems like we'd want if/else here instead of nested ternaries